_pages/home.md

```diff
@@ -14,24 +14,23 @@ SIGDAT is the [Association for Computational Linguistics](http://aclweb.org) spe
 
 <h2>About</h2>
 
-SIGDAT was founded in 1993 and is one of ACL's oldest SIGs. Since its inception, SIGDAT's primary mission has been to organize a series of conferences and workshops, including EMNLP (Conference on Empirical Methods in Natural Language Processing) and WVLC (Workshop on Very Large Corpora). These meetings have become quite popular, and and EMNLP is now a 3-day conference with 700+ attendees and 1300 paper submissions in 2015.
+SIGDAT was founded in 1993 and is one of ACL's oldest SIGs. Since its inception, SIGDAT's primary mission has been to organize a series of conferences and workshops, including EMNLP (Conference on Empirical Methods in Natural Language Processing) and WVLC (Workshop on Very Large Corpora). These meetings have become quite popular, and EMNLP is now a 3-day main conference with 2 day workshops and tutorials, having about 2500 attendees and 2137 paper submissions in 2018
```

Comment:
You are missing a period. It should be:

```suggestion
SIGDAT was founded in 1993 and is one of ACL's oldest SIGs. Since its inception, SIGDAT's primary mission has been to organize a series of conferences and workshops, including EMNLP (Conference on Empirical Methods in Natural Language Processing) and WVLC (Workshop on Very Large Corpora). These meetings have become quite popular, and EMNLP is now a 3-day main conference with 2 day workshops and tutorials, having about 2500 attendees and 2137 paper submissions in 2018.
```